providers/src/airflow/providers/edge/worker_api/routes/_v2_routes.py

```diff
@@ -87,3 +88,43 @@ def set_state_v2(worker_name: str, body: dict[str, Any], session=NEW_SESSION) ->
         return set_state(worker_name, request_obj, session)
     except HTTPException as e:
         return e.to_response()  # type: ignore[attr-defined]
+
+
+def logfile_path_v2(
+    dag_id: str,
+    task_id: str,
+    run_id: str,
+    try_number: int,
+    map_index: str,  # Note: Connexion can not have negative numbers in path parameters, use string therefore
```

Comment:
Do you want to add TODO to remove it once we drop Connexion?

Reply:
Did not we already drop it  and switched to fast_api ?

Reply:
Yes on main/Airflow 3 we switched to FastAPI - so do I - but Airflow 2.10 is still with Connexion. So the reword to move-off internal API is to provide the Edge entrypoints as REST API - and for the support of Airflow 2 it is still Connexion. I wanted to prevent the need / additional dependency to integrate FastAPI just for Edge into Airflow 2 when Connexion is already there and working. (Besides this small but in Connexion which seems to be inherited from Flask - no support to accept integer with a negative sign in front)